packages/editor-preview/src/browser/editor-preview-manager.ts

```diff
@@ -84,6 +84,12 @@ export class EditorPreviewManager extends WidgetOpenHandler<EditorPreviewWidget|
             this.shell.activateWidget(editorWidget.id);
             this.currentEditorPreview = Promise.resolve(undefined);
         });
+
+        this.preferences.onPreferenceChanged(change => {
```

Comment:
Should not we dispose a listener at some point of time? Otherwise we keep creating them for each new preview, but never release. Or move it into `init` section and implement based on `this.currentEditorPreview`.